fix.go

```diff
@@ -1,135 +0,0 @@
-package kubeaudit
```

Comment:
Why is this file removed?

Reply:
omg! I didn't mean to delete it! 😩I'll bring it back!